nginx/nginx.conf

```diff
@@ -96,13 +96,19 @@ http {
             root /media/frigate;
         }
 
-        location / {
+        location /api/ {
+            add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
_note: temporary until we come up with a better way to allow the dev environment access to the api … unless we want to just open this up_